ynput/ayon-ftrack
# Converted local delivery action

## Description
Converted local action to run delivery action. Fixed how entities are queried and changed how anatomy delivery templates are received.

## How to test
1. Upload addon to server, or make sure 1.0.0 version is on server when dev mode is used
2. Make sure your bundle is using ftrack addon
3. Start ayon launcher
4. Select published version/s on ftrack
5. Show actions for the selection
6. Trigger `Delivery` action
7. Fill form
8. Confirm
9. Wait until finished
